src/components/navigation/header/config/DrawerNodeItems.tsx

```diff
@@ -1271,32 +1271,13 @@ export const drawerNodeItems = ({
           translationKey: 'header.userMenu.contactDetails',
           link: {
             de: '/de/profile',
-            en: '/en/profile',
+            en: '/de/profile',
```

Comment:
Is there a reason to reset selected language here? I believe not since the new page supports English.

Reply:
Nope. I forgot that it was changed to en for privates already.